core/bin/core_api.rs

```diff
@@ -65,308 +66,12 @@ use dust::{
-                let qdrant_clients = self.qdrant_clients.clone();
-                let manager = self.run_manager.clone();
-
-                // Start a task that will run the app in the background.
-                tokio::task::spawn(async move {
-                    let now = std::time::Instant::now();
-
-                    match app
-                        .0
-                        .run(
-                            app.1,
-                            app.2,
-                            store,
-                            databases_store,
-                            qdrant_clients,
-                            None,
-                            app.3,
-                        )
-                        .await
-                    {
-                        Ok(()) => {
-                            info!(
-                                run = app.0.run_ref().unwrap().run_id(),
-                                app_version = app.0.hash(),
-                                elapsed = now.elapsed().as_millis(),
-                                "Run finished"
-                            );
-                        }
-                        Err(e) => {
-                            error!(error = %e, "Run error");
-                        }
-                    }
-                    {
-                        let mut manager = manager.lock();
-                        manager
-                            .pending_runs
-                            .retain(|run_id| run_id != app.0.run_ref().unwrap().run_id());
-                    }
-                });
-            });
-            loop_count += 1;
-            tokio::time::sleep(std::time::Duration::from_millis(4)).await;
-            if loop_count % 1024 == 0 {
-                let manager = self.run_manager.lock();
-                let runs_count = manager.pending_runs.len();
-                if runs_count > 0 || loop_count % 65536 == 0 {
-                    info!(pending_runs = runs_count, "Pending runs {}", runs_count);
-                }
-            }
-            // Roughly every 4 minutes, cleanup dead SQLite workers if any.
-            if loop_count % 65536 == 0 {
-                let store = self.store.clone();
-                tokio::task::spawn(async move {
-                    match store
-                        .sqlite_workers_cleanup(client::HEARTBEAT_INTERVAL_MS)
-                        .await
-                    {
-                        Err(e) => {
-                            error!(error = %e, "Failed to cleanup SQLite workers");
-                        }
-                        Ok(_) => (),
-                    }
-                });
-            }
-        }
-    }
-}
-
 /// Index
 
 async fn index() -> &'static str {
     "dust_api server ready"
 }
 
-/// Create a new project (simply generates an id)
-
-async fn projects_create(State(state): State<Arc<APIState>>) -> (StatusCode, Json<APIResponse>) {
```

Comment:
moved to `src/api/projects.rs`

could have been `src/api/handlers/projects.rs`